app/src/test/java/tttTDD/Java/BoardTest.java

```diff
@@ -33,17 +35,32 @@ public void getBoardSpace() {
     }
 
     @Test
-    public void updateBoardSpaceWithNewValue() throws Exception {
+    public void checkIfLocationIsOutOfBounds() {
+        board = new Board(3);
+
+        Assert.assertNull(board.getSpace(9));
+        Assert.assertNull(board.getSpace(16));
+    }
+
+    @Test
+    public void updateBoardSpaceWithNewValue(){
         board = new Board();
-        board.updateBoard(2, "X");
 
+        Assert.assertTrue(board.updateBoard(2, "X"));
         Assert.assertEquals(board.getSpace(2), "X");
```

Comment:
Is there any way to refactor the `Board` class such that the `Assert` section doesn't need to make any more calls to the board instance to prove this?

Reply:
The other way would be to compare the board data against an expected board state but we got rid of the method that calls `getBoard` because it had access to too much information per a conversation with @joeyshipley.